api-spec/extensions/fields.fragment.yaml

```diff
@@ -26,14 +26,13 @@ components:
           $ref: '#/components/schemas/fields'
     fields:
       description: |
-        The geometry member determines whether the geometry is populated or is null. The
-        include and exclude members specify an array of property names that are either 
-        included or excluded from the result, respectively. If both include and exclude 
-        are specified, include takes precedence.
+        The include and exclude members specify an array of
+        property names that are either included or excluded
+        from the result, respectively. If both include and
+        exclude are specified, include takes precedence.
+        Values should include the full JSON path of the property.
```

Comment:
Are we supporting the 'dot' notation only?

Reply:
That was true before this change for nested fields. It might be worth mentioning though.